views/marts/product/fct_active_users.view.lkml

```diff
@@ -76,31 +76,86 @@ view: fct_active_users {
   dimension: daily_desktop_active_users {
     type: number
     sql: ${TABLE}.daily_desktop_active_users ;;
-    label: "Desktop/Client DAU"
-    description: "DAU reported by Rudderstack telemetry."
+    label: "Desktop DAU"
+    description: "Desktop DAU reported by telemetry."
   }
 
   measure: total_daily_desktop_active_users {
     type: sum
     sql: ${daily_desktop_active_users} ;;
-    label: "Total Desktop/Client DAU"
+    label: "Total Desktop DAU"
     view_label: "  * Metrics. DAU"
   }
 
-  dimension: daily_legacy_active_users {
+  dimension: daily_webapp_active_users {
     type: number
-    sql: ${TABLE}.daily_legacy_active_users ;;
-    label: "Legacy Desktop/Client DAU"
-    description: "DAU reported by Segment telemetry."
+    sql: ${TABLE}.daily_webapp_active_users ;;
+    label: "Webapp DAU"
+    description: "Webapp DAU reported by telemetry."
+  }
+
+  measure: total_daily_webapp_active_users {
+    type: sum
+    sql: ${daily_webapp_active_users} ;;
+    label: "Total Webapp DAU"
+    view_label: "  * Metrics. DAU"
   }
 
-  measure: total_daily_legacy_active_users {
+  dimension: daily_client_desktop_active_users {
```

Comment:
Is this different than `daily_desktop_active_users`? If not, makes sense to keep only one of the two.

Reply:
this is different, we have 2 cohorts split by 2 sources.
`client_desktop` -> Rudderstack and Desktop
`legacy_desktop` -> Segment and Desktop
`client_webapp` -> Rudderstack and Webapp
`legacy_webapp` -> Segment and Webapp
and
`desktop_active_users` -> Rudderstack & Segment Desktop
`webapp_active_users` -> Rudderstack & Segment Webapp
and
`daily_active_users` -> Rudderstack & Segment Desktop & Webapp